test/hardhat/irm/Irm.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { AbiCoder, keccak256, toBigInt } from "ethers";
+import hre from "hardhat";
+import _range from "lodash/range";
+import { AdaptativeCurveIrm } from "types";
```

Comment:
```suggestion
import { AdaptiveCurveIrm } from "types";
```
same change below multiple times